.github/CODEOWNERS

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 ADBC Drivers Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+* @lidavidm @zeroshade
+.github/* @lidavidm @zeroshade
```

```suggestion
/.github/ @lidavidm @zeroshade
```

I'm not sure I totally get what works and what doesn't with CODEOWNERS but on the Arrow repos  [[1]](https://github.com/apache/arrow/blob/main/.github/CODEOWNERS#L51)[[2]](https://github.com/apache/arrow-adbc/blob/main/.github/CODEOWNERS) we use this pattern. `*` is "everything but a slash" so maybe the other rules in this PR should be tweaked too? I tried a minimal test on a throwaway repo and couldn't even get a basic test working so I'm not sure.